tests/api.c

```diff
@@ -23879,15 +23879,15 @@ static void test_wc_PKCS7_EncodeSignedData(void)
     PKCS7*      pkcs7;
     WC_RNG      rng;
     byte        output[FOURK_BUF];
-    byte        badOut[0];
+    byte        badOut[1];
```

Comment:
Would it not be better to change it to `badOut[]` or does pedantic gcc complain about that as well?

Reply:
Not pedantic, but no size defined.
```
tests/api.c: In function ‘test_wc_PKCS7_EncodeSignedData’:
tests/api.c:23882:17: error: array size missing in ‘badOut’
23882 |     byte        badOut[];
      |                 ^~~~~~
```

And if I try empty curlies I also get this:
```
tests/api.c: In function ‘test_wc_PKCS7_VerifySignedData’:
tests/api.c:24363:23: error: ISO C forbids empty initializer braces [-Werror=pedantic]
24363 |     byte   badOut[] = {};
      |                       ^
```